ddmrp/models/procurement_rule.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp (https://www.camptocamp.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ProcurementRule(models.Model):
+    _inherit = 'procurement.rule'
+
+    def _prepare_mo_vals(self, product_id, product_qty, product_uom,
+                         location_id, name, origin, values, bom):
+        result = super(ProcurementRule, self)._prepare_mo_vals(
+            product_id, product_qty, product_uom, location_id,
+            name, origin, values, bom
+        )
+        # this field can be passed by
+        # StockWarehouseOrderpoint._prepare_procurement_values
+        # (yes as a recordset!)
+        if values.get('orderpoint_id'):
+            result['orderpoint_id'] = values['orderpoint_id'].id
+        return result
+
+    def _run_manufacture(self, product_id, product_qty, product_uom,
+                         location_id, name, origin, values):
+        super(ProcurementRule, self)._run_manufacture(
+            product_id, product_qty, product_uom,
+            location_id, name, origin, values
+        )
+        orderpoint = values.get('orderpoint_id')
```

Comment:
aren't you changing `orderpoint_id` from recordset to ID right above?

Reply:
The change above is unrelated to the values we get here :
```
> /opt/odoo/external-src/ddmrp/ddmrp/models/procurement_rule.py(30)_run_manufacture()
-> orderpoint = values.get('orderpoint_id')
(Pdb) values
{'group_id': procurement.group(), 'date_planned': '2018-04-10 11:38:37', 'orderpoint_id': stock.warehouse.orderpoint(119,), 'company_id': res.company(1,), 'warehouse_id': stock.warehouse(1,), 'priority': '1'}
```